examples/skywalking-tracing/service1-envoy-skywalking.yaml

```diff
@@ -0,0 +1,134 @@
+                route:
+                  cluster: local_service
+                decorator:
+                  operation: checkAvailability
+          http_filters:
+          - name: envoy.filters.http.router
+            typed_config:
+              "@type": type.googleapis.com/envoy.extensions.filters.http.router.v3.Router
+              start_child_span: true
+  - address:
+      socket_address:
+        address: 0.0.0.0
+        port_value: 9000
+    traffic_direction: OUTBOUND
+    filter_chains:
+    - filters:
+      - name: envoy.filters.network.http_connection_manager
+        typed_config:
+          "@type": type.googleapis.com/envoy.extensions.filters.network.http_connection_manager.v3.HttpConnectionManager
+          tracing:
+            provider:
+              name: envoy.tracers.skywalking
+              typed_config:
+                "@type": type.googleapis.com/envoy.config.trace.v3.SkyWalkingConfig
+                grpc_service:
+                  envoy_grpc:
+                    cluster_name: skywalking
+                  timeout: 0.250s
+                client_config:
+                  service_name: service1-envoy
+                  instance_name: service1-envoy-1
+          codec_type: auto
+          stat_prefix: egress_http
+          route_config:
+            name: service2_route
+            virtual_hosts:
+            - name: service2
+              domains:
+              - "*"
+              routes:
+              - match:
+                  prefix: "/trace/2"
+                route:
+                  cluster: service2
+                decorator:
+                  operation: checkStock
+          http_filters:
+          - name: envoy.filters.http.router
+            typed_config:
+              "@type": type.googleapis.com/envoy.extensions.filters.http.router.v3.Router
+              start_child_span: true
+  clusters:
+  - name: local_service
+    connect_timeout: 0.250s
+    type: strict_dns
+    lb_policy: round_robin
+    load_assignment:
+      cluster_name: local_service
+      endpoints:
+      - lb_endpoints:
+        - endpoint:
+            address:
+              socket_address:
+                address: 127.0.0.1
+                port_value: 8080
+  - name: service2
+    connect_timeout: 0.250s
+    type: strict_dns
+    lb_policy: round_robin
+    http2_protocol_options: {}
+    load_assignment:
+      cluster_name: service2
+      endpoints:
+      - lb_endpoints:
+        - endpoint:
+            address:
+              socket_address:
+                address: service2
+                port_value: 8000
+  - name: skywalking
+    connect_timeout: 1s
+    type: strict_dns
+    lb_policy: round_robin
+    http2_protocol_options: {}
+    load_assignment:
+      cluster_name: skywalking
+      endpoints:
+      - lb_endpoints:
+        - endpoint:
+            address:
+              socket_address:
+                address: skywalking-oap
+                port_value: 11800
+admin:
```

Comment:
the same comment as about the frontend `admin` - if its not used/exposed we should remove